test/verify/check-login

```diff
@@ -343,6 +343,28 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         b.wait_not_visible("#login")
         b.wait_not_visible("#login-details")
 
+    @skipImage("Don't run upstream tests against distro packages", "rhel-8-0-distropkg")
```

Comment:
Please replace the reason with "added in PR #12493"

Reply:
Done.